samples/sensor/accel_polling/sample.yaml

```
@@ -3,26 +3,15 @@ sample:
 tests:
   sample.sensor.accel_polling:
     harness: console
-    tags: sensors
     filter: dt_alias_exists("accel0")
+    tags:
+      - sensors
+      - accelerometer
     harness_config:
       type: one_line
       regex:
         - "^\\s*[0-9A-Za-z_,+-.]*@[0-9A-Fa-f]* \\[m\/s\\^2\\]:    \
            \\(\\s*-?[0-9\\.]*,\\s*-?[0-9\\.]*,\\s*-?[0-9\\.]*\\)$"
     integration_platforms:
-      - cc1352r_sensortag               # adxl362
-      - blueclover_plt_demo_v2/nrf52832 # bmi270
-      - frdm_kl25z                      # mma8451q
-      - lpcxpresso55s28                 # mma8652fc
-      - bbc_microbit                    # mmc8653fc
-      - frdm_k64f                       # fxos8700
-      - mimxrt1040_evk                  # fxls8974
-      - sparkfun_thing_plus/nrf9160     # lis2dh
-      - thingy52/nrf52832               # lis2dh12
-      - stm32f411e_disco                # lsm303agr_accel
       - stm32f3_disco                   # lsm303dlhc_accel
-      - bl5340_dvk/nrf5340/cpuapp       # lis3dh
-      - b_l4s5i_iot01a                  # lsm6dsl
-      - sensortile_box                  # lis2dw12, lsm6dso, iisdhhc
-      - thingy53/nrf5340/cpuapp         # adxl362, bmi270
+      - cc1352r_sensortag               # adxl362
```

This isn't trying to list all supported boards; it's selecting set of boards that cover different accelerometer drivers. Some duplicates slipped in along the way. This should be:
```suggestion
      - frdm_kl25z                      # mma8451q
      - lpcxpresso55s28                 # mma8652fc
      - bbc_microbit                    # mmc8653fc
      - frdm_k64f                       # fxos8700
      - mimxrt1040_evk                  # fxls8974
      - sparkfun_thing_plus/nrf9160     # lis2dh
      - thingy52/nrf52832               # lis2dh12
      - stm32f411e_disco                # lsm303agr_accel
      - stm32f3_disco                   # lsm303dlhc_accel
      - bl5340_dvk/nrf5340/cpuapp       # lis3dh
      - b_l4s5i_iot01a                  # lsm6dsl
      - sensortile_box                  # lis2dw12, lsm6dso, iisdhhc
      - thingy53/nrf5340/cpuapp         # adxl362, bmi270
```

---

integration_platforms is supposed to list a small subset (1 or 2) for coverage and not be an exhaustive list of anything that can build, which is the case here. All supported platform will then be covered used the filter. 

see https://docs.zephyrproject.org/latest/samples/sample_definition_and_criteria.html#twister-should-be-able-to-build-every-sample

"Make use of integration_platforms to limit the scope when there are timing or resource constraints. Ideally, one platform should be enough to verify the sample when changes are submitted to the Zephyr tree via a pull-request."

also

https://docs.zephyrproject.org/latest/develop/test/twister.html

> integration_platforms: <YML list of platforms/boards>
This option limits the scope to the listed platforms when twister is invoked with the --integration option. Use this instead of platform_allow if the goal is to limit scope due to timing or resource constraints.

We are not verifying platforms here, we are verifying the sample and that does not need to be done on each platform with this sensor.

---

Not trying to verify platforms. We're verifying the sample builds for different sensors (that's what the comment after each platform is about)

---

right, but integration_platforms is about limiting the scope, not adding every single possible hardware, the coverage and inclusion for the various sensors should be done in some other way